### ISSUE TITLE
VC: Google Maps widget doesn't function when Ultimate Addons for WPBakery Page Builder is activated

### ISSUE THREAD

**Issue by Misplon** [siteorigin/so-widgets-bundle#862]
The Google Maps widget doesn't function correctly when Ultimate Addons for WPBakery Page Builder plugin is activated. The problem persists with all the modules in Ultimate Addons for WPBakery Page Builder deactivated. It isn't possible to search for a location as the map center. 

```Uncaught (in promise) TypeError: $ is not a function
    at Object.sowbForms.setupLocationFields (location-field.min.js:1)
    at sowbAdminGoogleMapInit (location-field.min.js:1)
    at maps.googleapis.com/maps/api/js?key=&libraries=places&callback=sowbAdminGoogleMapInit:131
    at maps.googleapis.com/maps/api/js?key=&libraries=places&callback=sowbAdminGoogleMapInit:131```

**Comment by Misplon** [comment]
@braamgenis Can you push the hardening changes we discussed on Slack so I can test?

**Comment by braamgenis** [comment]
Fixed in 5b76622a8e0398b983b57d06ed7e8be735e48536
